packages/protocol/lib/compatibility/verify-bytecode.ts

```diff
@@ -0,0 +1,277 @@
+import { linkLibraries, stripMetadata } from '@celo/protocol/lib/bytecode'
+import { ProposalTx } from '@celo/protocol/scripts/truffle/make-release'
+import { ProxyInstance, RegistryInstance } from 'types'
+
+import { BuildArtifacts, } from '@openzeppelin/upgrades'
+
+import Web3 from 'web3'
+
+const ignoredContracts = [
```

Comment:
does `MetaTransactionWallet` need to be added here, or is that different?

Reply:
This is only to filter out contracts that appear in the `CeloContractName` enum in `lib/registry-utils`. MetaTransactionWallet is a contract that would be deployed on a per-user basis - it's not a part of the core Celo system.